### DISCUSSION TITLE
Skip reads correction step in metaSPAdes

### DISCUSSION THREAD

**Discussion by mchialva** [ablab/spades#1135] [Q&A]
Hi, 
I'm trying to assemble complex soil metagenomes (2x150, Novaseq) at a relatively high sequencing depth using metaSPAdes and I was wondering if it can be worth to skip the reads corrections step. I read somewhere in previous Q&A here that reads correction in large datasets, beside increasing computational time, tend to collapse similar sequences. I tried both option on a test metagenome and found that without reads correction the contig number increased of four times. What would you recommend on this point?
Thanks

**Comment by asl** [comment]
Well, absolute contig number frankly speaking does not mean anything. As SPAdes does not do any filtering of the output, the extra contigs might be just some uncorrected stuff.

I would recommend you to analyze the results in terms of the contents, length, etc.
